pkgs/tools/system/bpytop/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bpytop";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "aristocratos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0cxyrk5a9j0ymll9h5b6jq48yjy9srcxh4rmsqk8w0d14prmflgg";
+  };
+
+  phases = [ "unpackPhase" "fixupPhase" "installPhase" ];
```

Comment:
Why though? Switching the order of phases is rather harmful IMHO. Also I wouldn't recommend to override `fixupPhase`. IMHO placing the `sed`-expr in `postPatch` and setting `dontBuild = true;` should be enough.

Reply:
Oh… I think `dontBuild` fell victim to cache eviction. m(